hindi/king/king.md

```diff
@@ -0,0 +1,5 @@
+Once upon a time there was a great king in the far away galaxy called "Swarg".
+And then one day he came to another galaxy called "Milky Way" on a vocation.
+After long wondering he found earth...
+and now days he is busy in writing stories on GitHub.
+To read more about him click [Here](www.google.com) and search for him.
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.